### DISCUSSION TITLE
Multi key press/character keybinding (e.g. jj to hide completion menu)

### DISCUSSION THREAD

**Discussion by abzrgzdh** [saghen/blink.cmp#747] [Q&A]
I tried it and it did not do anything. It is something I use a lot when I edit tex files. It is possible to define such a keybinding in nvim.cmp. Am I missing sth or it is not implemented?

**Comment by abzrgzdh** [comment]
Does anybody have any idea how it can be done?

**Reply by krovuxdev** [reply]
@abzrg, have you tried using `require("blink.cmp").hide()`? If you assign it to a Neovim keymap, does it work for you?

**Reply by abzrgzdh** [reply]
Aha. Very nice! Thank you @krovuxdev.
